tests/stress_tests/common/tests_utils.h

```diff
@@ -63,11 +63,11 @@ class MemLeaksTestCase : public TestCaseBase {
                      std::vector<std::map<std::string, std::string>> _models) {
         numprocesses = _numprocesses, numthreads = _numthreads, numiters = _numiters,
         device = _device, models = _models;
-        test_case_name = "Numprocesses_" + std::to_string(numprocesses) + "_Numthreads_" + std::to_string(numthreads) +
-                         "_Numiters_" + std::to_string(numiters) + "_Device_" + update_item_for_name(device);
+        test_case_name = "NP" + std::to_string(numprocesses) + "_NT" + std::to_string(numthreads) +
+                         "_NI" + std::to_string(numiters) + safe_string(device);
         for (size_t i = 0; i < models.size(); i++) {
-            test_case_name += "_Model" + std::to_string(i + 1) + "_" + update_item_for_name(models[i]["name"]) + "_" +
-                              update_item_for_name(models[i]["precision"]);
+            test_case_name += "_Model" + std::to_string(i + 1) + "_" + safe_string(models[i]["name"]) + "_" +
+                              safe_string(models[i]["precision"]);
```

Comment:
These changes are not relevant anymore, but I kept them since they don't break anything.